File: AngularEducationStore/src/app/shared/models/printing-editions/PrintingEditionModel.ts
import { BaseModel } from 'src/app/shared/models/base/BaseModel';
import { PrintingEditionModelItem } from 'src/app/shared/models/printing-editions/PrintingEditionModelItem';

export class PrintingEditionModel extends BaseModel {
    public items = new Array<PrintingEditionModelItem>();
}

File: EducationApp.BusinessLayer/Models/PrintingEditions/PrintingEditionModel.cs
using EducationApp.BusinessLogic.Models.Base;
using System.Collections.Generic;

namespace EducationApp.BusinessLogic.Models.PrintingEditions
{
    public class PrintingEditionModel : BaseModel
    {
        public ICollection<PrintingEditionModelItem> Items = new List<PrintingEditionModelItem>();

    }
}
